benchmarks.yaml

```diff
@@ -143,20 +126,20 @@
   server: Actix Web
   run:
     cmd: ./target/release/juniper
-- id: sangria
-  name: Sangria
-  url: https://github.com/sangria-graphql/sangria
-  lang: Scala
-  server: Akka HTTP
-  run:
-    cmd: java
-    args:
-      [
-        "-Xrs",
-        "-Xmx4G",
-        "-jar",
-        "./target/scala-2.13/sangria-assembly-0.1.0-SNAPSHOT.jar",
-      ]
```

Comment:
Why should we disable sangria?

Reply:
[Action](https://github.com/graphql-crystal/benchmarks/actions/runs/6730873449/job/18294413019#step:9:502)

Check this one, it just halts there on sangria

Reply:
@jgillich could comment on it.